signup-service/src/main/java/io/mosip/signup/services/WebSocketHandler.java

```diff
@@ -158,6 +164,7 @@ private void handleVerificationResult(IdentityVerifierPlugin plugin, IdentityVer
             log.error("Failed to fetch verified result from the plugin", e);
             transaction.setStatus(VerificationStatus.FAILED);
             transaction.setErrorCode(e.getErrorCode());
+            auditHelper.sendAuditTransaction(AuditEvent.PROCESS_FRAMES, AuditEventType.ERROR,transaction.getSlotId() , null);
         } catch (ProfileException e) {
             log.error("Failed to update profile", e);
             transaction.setStatus(VerificationStatus.FAILED);
```

Comment:
Why is the ERROR audit not raised from the ProfileException catch block? Can we create single catch block to handle both the exceptions? @pvsaidurga